Integrations/TinkoffIntegrationLib/TinkoffInstrumentAdapter.cs

```diff
@@ -28,7 +28,9 @@ public TinkoffInstrumentAdapter(InstrumentType type, MarketInstrument instrument
 
         public override string Name => tinkoffInstrument.Name;
 
-        public override string Currency => tinkoffInstrument.Currency.ToString();
+        public override DTO.MarketBrokerObjects.Currency Currency 
```

Comment:
Why not use `using...`?

Reply:
Because with using by default choses tinkoff currency